src/client.rs

```diff
@@ -257,6 +244,12 @@ pub struct ClientStats {
     pub indexes: HashMap<String, IndexStats>,
 }
 
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Health {
+    pub status: Option<String>,
```

Comment:
It would be great to provide some basic documentation about `Health` and the value of its field.